manifests/entandoapps.entando.org.crd.yaml

```diff
@@ -74,44 +58,33 @@ spec:
                     Operator will create an Ingress reflecting this hostname, and expose this EntandoApp on the
                     path specified in 'EntandoApp.spec.ingressPath' on
-                    this host. If omitted, the Entando Operator will automatically generate a hostname using the value of
-                    the ENTANDO_DEFAULT_ROUTING_SUFFIX environment variable that the Entando Operator was configured with.
-                    It will prefix the name and namespace of the EntandoApp to this default routing suffix.
+                    this host. If omitted, the Entando Operator will automatically generate a hostname.
+
+                    See [here] (https://dev.entando.org/v6.3/docs/concepts/operator-intro.html#ingress-path-generation) for how the operator will create and configure the Ingress.
                   type: string
                   pattern: '^([A-Za-z0-9-]{1,63}\.)*[[A-Za-z0-9-]{1,63}$'
                 ingressPath:
                   description: >
                     The path that will be used to expose this EntandoApp on the ingress host
                     specified in 'EntandoApp.spec.ingressHostname'. This property needs to be in sync with
-                    the web context path that is used in the underlying WAR deployment. It should there either be
+                    the web context path that is used in the underlying WAR deployment. It should either be
                     the name of the WAR file that gets generated, or it needs to be in sync with the web context
                     specified in the jboss-web.xml deployment descriptor. The default value is 'entando-de-app'.
                   type: string
                 tlsSecretName:
                   description: >
-                    The name of a standard Kubernetes TLS Secret, assumed to be in the same Namespace as the EntandoApp
-                    being created. This Secret is expected to have a private key, and a certificate for the hostname that
-                    the EntandoApp will be exposed on. Please refer to the 'ingressHostname' property for more
-                    information on how the hostname is determined. If absent, the Entando Operator will evaluate the
-                    value of the ENTANDO_PATH_TO_TLS_KEYPAIR which is expected to contain two files: tls.key and tls.crt.
-                    If a keypair is found in the folder specified, it will revert to the keypair found. Alternatively,
-                    the Entando Operator will evaluate the value of the ENTANDO_USE_AUTO_CERT_GENERATION. If set to 'true'
-                    the Entando Operator will assume that the cluster has been configured with a valid CA and leave it to
-                    the Ingress controller in use to generate its own certificates.
+                    The name of a standard Kubernetes TLS Secret, assumed to be in the same Namespace as the EntandoApp being created.
```

Comment:
Is it in the same namespace in the case of  the cluster-scoped operator?  I think the tutorial had the secrets in the openshift-operator namespace but I don't know if the secrets get copied over as well?